networkit/cpp/linkprediction/test/LinkPredictionGTest.cpp

```diff
@@ -242,4 +245,69 @@ TEST_F(LinkPredictionGTest, testKatzRunOnOrdering) {
     }
 }
 
+TEST_F(LinkPredictionGTest, testAdamicAdarIndex) {
+    METISGraphReader graphReader;
+    Graph G = graphReader.read("input/jazz.graph");
+
+    AdamicAdarIndex AAI(G);
+    std::vector<NetworKit::LinkPredictor::prediction> preds = AAI.runAll();
+    for (auto const &pred : preds) {
+        // check for each pair of nodes that no edge exists
+        EXPECT_FALSE(G.hasEdge(pred.first.first, pred.first.second));
+        // expect each value to be smaller than the number of common neighbours
+        auto commonNeighbours =
```

Comment:
For this and the other occasions: Better use American English (`neighbour` -> `neighbor`).